src/composables/auth.ts

```diff
@@ -0,0 +1,117 @@
+import { BiometricAuth } from '@aparajita/capacitor-biometric-auth';
+import {
+  ref,
+  readonly,
+  watch,
+  effectScope,
+} from 'vue';
+
+import { tg as t } from '@/popup/plugins/i18n';
+import { IS_MOBILE_APP, MODAL_ENABLE_SECURE_LOGIN, MODAL_SECURE_LOGIN } from '@/constants';
+import { useUi } from './ui';
+import { useModals } from './modals';
+
+let composableInitialized = false;
+const isAvailable = ref(false);
+const isAuthenticated = ref(false);
+
+export function useAuth() {
+  const { isSecureLoginEnabled } = useUi();
+  const { openModal } = useModals();
+
+  async function checkIsAvailable() {
+    isAvailable.value = (await BiometricAuth.checkBiometry()).isAvailable;
+  }
```

Comment:
I suggest to use similar approach as `getAeSdk` so we set the value only on first call and then return it.